client/coral-admin/src/routes/Moderation/components/Moderation.js

```diff
@@ -7,6 +7,7 @@ import ModerationMenu from './ModerationMenu';
 import ModerationHeader from './ModerationHeader';
 import ModerationKeysModal from '../../../components/ModerationKeysModal';
 import StorySearch from '../containers/StorySearch';
+import {isPremod, getModPath} from '../../../utils';
 
 export default class Moderation extends Component {
```

Comment:
I need this to use it on the `componentWillMount` here because in the container I don't have `root` data populated. :/